src/Mechanisms/FrontendAssets/FrontendAssets.php

```diff
@@ -64,12 +68,16 @@ public static function livewireStyles($expression)
 
     public function returnJavaScriptAsFile()
     {
-        return Utils::pretendResponseIsFile(__DIR__.'/../../../dist/livewire.js');
+        return Utils::pretendResponseIsFile(
+            config('app.debug')
+                ? __DIR__.'/../../../dist/livewire.js'
+                : __DIR__.'/../../../dist/livewire.min.js'
+        );
     }
 
     public function maps()
     {
-        return Utils::pretendResponseIsFile(__DIR__.'/../../../dist/livewire.js.map');
+        return Utils::pretendResponseIsFile(__DIR__.'/../../../dist/livewire.min.js.map');
```

Comment:
@joshhanley Doesn't this again need a `config('app.debug') ? ... : ...`?

Reply:
@levu42 There's no map for a non-minified file, so I didn't think it was needed. But I could be wrong 😬

Reply:
@joshhanley ah, I see! I didn't know the code before, just looked at the diff and thought you might have overlooked something :)